### ISSUE TITLE
Type error in create-svelte typescript default template

### ISSUE THREAD

**Issue by SaeedYasin** [sveltejs/kit#4787]
### Describe the bug

Created default todos app using typescript and got this error without making any code changes, I'll submit a PR.

<img width="1128" alt="Error" src="https://user-images.githubusercontent.com/26478201/166101659-16596bc1-0585-4243-881a-28f27531e6a7.png">

### Reproduction

Run, npm init svelte@next svelte-todo
Choose Sveltekit Demo App, typescript, eslint, prettier and playwright.
Open svelte-todo code in vscode and goto src/routes/todos/index.ts and observe the error on line 2.

### System Info

```shell
System:
    OS: macOS 12.3.1
    CPU: (10) arm64 Apple M1 Pro
    Memory: 1.02 GB / 32.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.14.2 - /usr/local/bin/node
    npm: 8.5.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 100.0.4896.127
    Safari: 15.4
```

### Severity

annoyance

**Comment by dummdidumm** [comment]
Pending fix for this in #4705 already, although we're not sure if we will merge as is or rethink where the types are imported from first.

**Comment by SaeedYasin** [comment]
@dummdidumm alright, sounds like there is no need for this issue. I'll close it then. Thanks.
